app/views/user_credentials/index.html.erb

```diff
@@ -39,6 +39,7 @@
       <div class="btn-group">
         <%= link_to new_user_credential_path, class: "btn btn-default" do %><span class="glyphicon glyphicon-plus"></span> New Credential<% end %>
         <%= link_to user_credentials_path(format: :json), class: "btn btn-default" do %><span class="glyphicon glyphicon-cloud-download"></span> Download Credentials<% end %>
+        <%= form_tag import_user_credentials_path, multipart: true, class: "btn btn-default" do %><span class="glyphicon glyphicon-upload"><%= file_field_tag :file %><%= submit_tag "Upload File"%></span><% end %>
       </div>
```

Comment:
No they don't but I found a small solution and now it looks better

![screen shot 2016-04-13 at 12 05 37 pm](https://cloud.githubusercontent.com/assets/1547934/14500471/4a6505ce-0170-11e6-8e20-1131c417c6c5.png)